README.md

```diff
@@ -10,79 +10,117 @@ REQUIREMENTS
 
 * PHP5
 * MySQL 5
-* MediaWiki 1.11+ 
+* MediaWiki 1.11+
 * phpBB 3.2
 
 INSTALL:
 =================
 
-Create a group in PHPBB for your wiki users. I named mine "Wiki". 
+Create a group in phpBB for your wiki users. I named mine "Wiki". 
 You will need to put the name you choose in the code below. 
 
-NOTE: In order for a user to be able to use the wiki they will need to 
+PHPBB GROUP NOTE: In order for a user to be able to use the wiki they will need to 
 be a member of the group you made in the step above.
 
-Put Auth_phpbb.php in /extensions/  
-Put iAuthPlugin.php in /extensions/  
-Put PasswordHash.php in /extensions/
+Put Auth_phpbb.php in /extensions/Auth_phpBB
+Put iAuthPlugin.php in /extensions/Auth_phpBB
+Put PasswordHash.php in /extensions/Auth_phpBB
 
+OPTIONAL PHPBB-TO-MEDIAWIKI USERNAME TRANSLATION FEATURE:
+phpBB usernames can be translated to more restrictive wiki usernames.
+Use phpBB ACP, Users and Groups, Custom profile fields.
+Create a "Single text field" custom profile field.
+Suggested settings:
+  Name it "wikiusername"
+  Set "Publicly display profile field" = no
+  Uncheck all visibility options, except check "Hide profile field" 
+  Set "Field name/title presented to the user" = "Wiki Username"
+  Set "Field description" = "Forum username translated for wiki username restrictions"
+  Set "Length of input box" = 20
+  Set "Maximum number of characters" = 255
+  Set "Field validation" to "Any character"
+  Set Language definitions fields to same as field name/title/description above.
+
+PROFILE NOTE:
+The custom profile field must be hidden to all but the admins
+because users could otherwise hijack wiki accounts by entering any
+username they wish.
+
+PROFILE NOTE:
+Enter a valid MediaWiki username into this field in the user's
+profile only when the phpBB username conflicts with MediaWiki username restrictions.
+For example, enter "Under Score" for a user with the name "Under_Score" because
+underscores are not allowed in MediaWiki usernames.  All users with phpBB usernames
+which are also valid MediaWiki usernames do not need this field set.
 
 Open LocalSettings.php. Put this at the bottom of the file. Edit as needed.
 
-        /*-----------------[ Everything below this line. ]-----------------*/
+/*-----------------[ Everything below this line. ]-----------------*/
         
-        // PHPBB User Database Plugin. (Requires MySQL Database)
-        require_once './extensions/Auth_phpbb.php';
+// phpBB User Database Plugin. (Requires MySQL Database)
+require_once '$IP/extensions/Auth_phpBB/Auth_phpbb.php';
```

Comment:
My mistake.  I just looked at our LocalSetting.php and I had not revised it yet from the relative path.  Single quotes failed.  Double quotes win.